lib/screens/user.dart

```diff
@@ -94,25 +98,97 @@ class UserScreenState extends State<UserScreen> {
           ],
         ),
       ),
+      
       floatingActionButton: Column(
```

Comment:
pretty sure this state != state of the whole page. so when you setState in the sub-widget flutter doesn't re render. i think you should probably create a stateful widget, with basically the content of this floatingActionButton thing and put it in there. that would give the sub widget its own state